### PR TITLE
New API proposal-Dedicated Networks

### LINKED ISSUE #60: New API Proposal - Dedicated Networks API
Fixed and Mobile Networks offer the capability of separating devices in different (logical) dedicated networks. Multiple of these (logical) dedicated networks can exist on the same physical network. Often, such a dedicated network with a specific performance (e.g. speed/ latency) is only needed for a specific time duration (e.g. one hour) and at specific locations (e.g. the area of a festival), potentially ensured by an SLA. The aim of the API is to request/modify/delete a dedicated network for API consumers with above mentioned characteristics. API consumers shall also be able to handle access to this dedicated network for devices. 

PR Available #59

### PR DESCRIPTION
**Pull request by tlohmar**
#### What type of PR is this?
New API Proposal

#### What this PR does / why we need it:

Fixed and Mobile Networks offer the capability of separating devices in different (logical) dedicated networks. 

#### Which issue(s) this PR fixes:

Fixes #60

### CODE REVIEW

File: documentation/API proposals/APIproposal_DedicatedNetworks.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+| **Field** | Description | 
+| ---- | ----- |
+| API family name | Dedicated Networks |
+| API family owner | Ericsson, Nokia, Telefonica, Vodafone |
```
Comment (jgarciahospital) [inline documentation/API proposals/APIproposal_DedicatedNetworks.md:4]:
```suggestion
| API family owner | Ericsson |
```

File: documentation/API proposals/APIproposal_DedicatedNetworks.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+| Technical viability | This API relies on a networking feature like the 5G NPN (specifically the PNI-NPN, TS 23.501 Cl 5.30) or the Network Communication Service (GSMA OPG.02 Annex H) which has several realization options such as Slices or APN/DNN. The API also allows to be applied to fixed or any other type of network scenarios. |
+| Commercial viability | The need for having a dedicated network with given performance  characteristics for an application is seen as a requirement / use case (?) by different industry verticals. For example, 5G-ACIA ([link](https://5g-acia.org/whitepapers/npns-for-industrial-scenarios/)) and 5G-MAG ([Link](https://www.5g-mag.com/post/5g-npns-for-media-production-collaboration-with-third-party-networks)) have release whitepapers, how dedicated networks can be used within industrial or live video production environments. |
+| YAML code available? | No |
+| Validated in lab/productive environments? | No |
+| Validated with real customers? | No |
+| Validated with operators? | No |
+| Supporters in API Backlog Working Group |  |
```
Comment (jgarciahospital) [inline documentation/API proposals/APIproposal_DedicatedNetworks.md:13]:
```suggestion
| Supporters in API Backlog Working Group | Nokia, Telefonica, Vodafone |
```

**Review by jgarciahospital — CHANGES_REQUESTED** [review]
Moving supporters to proper field

**Comment by hdamker** [comment]
Could this PR be merged? I'm already creating the repository for it as decided in last TSC.

**Review by jgarciahospital — APPROVED** [review]
LGTM

**Comment by hdamker** [comment]
@jgarciahospital I would just not close the issue #60 with the PR, as Codeowners and Maintainers are still collected there.

**Comment by jgarciahospital** [comment]
PR to be merged as API validated in TSC and group created with participants assigned (thanks @tlohmar). @camaraproject/api-backlog_maintainers can anyone review for merging?

**Comment by hdamker** [comment]
I need the link for the AHC ... therefore I will merge now.

@jgarciahospital Could we agree that API Proposals are merged BEFORE they are brought to the TSC? And do changes after the TSC with new PRs? That would create more transparency.

**Pull request merged**